.clang-tidy

```diff
@@ -1,10 +1,21 @@
 ---
-Checks:                '*,-clang-analyzer-alpha*,-llvmlibc-*,-google-*,-fuchsia-*,-hicpp-*,-bugprone-lambda-function-name,-modernize-use-trailing-return-type,-readability-braces-around-statements,-readability-named-parameter,-readability-uppercase-literal-suffix'
+Checks: >
+  *,
+  -clang-analyzer-alpha*,
+  -llvmlibc-*,
+  -google-*,
+  -fuchsia-*,
+  -hicpp-*,
+  -bugprone-lambda-function-name,
+  -modernize-use-trailing-return-type,
+  -readability-braces-around-statements,
+  -readability-named-parameter,
+  -readability-uppercase-literal-suffix,
+  -cppcoreguidelines-pro-type-reinterpret-cast
```

Comment:
We should sort the globs and non-globs alphabetically.

Reply:
Good idea.